thenewboston_node/business_logic/models/node.py

```diff
@@ -23,6 +27,35 @@ class Node(BaseDataclass):
         default=None, metadata={'example_value': '7a5dc06babda703a7d2d8ea18d3309a0c5e6830a25bac03e69633d283244e001'}
     )
 
+    @validates('node')
+    def validate(self):
+        self.validate_identifier()
+        self.validate_fee_amount()
+        self.validate_fee_account()
+        self.validate_network_addresses()
+
+    @validates('node identifier')
+    def validate_identifier(self):
+        validate_not_empty(f'{self.humanized_class_name} identifier', self.identifier)
+        validate_type(f'{self.humanized_class_name} identifier', self.identifier, str)
```

Comment:
I think that we should also validate that is contains only hexadecimal characters

Reply:
same for fee_account